### PR TITLE
added readme with funny image link

### PR DESCRIPTION
**Pull request by greydays**

### CODE REVIEW

File: greydays/readme.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This is my funny image:
+
+(https://lh6.googleusercontent.com/-mTMkTxm3_9I/USvaUXMlGVI/AAAAAAAAEzQ/Ziy5Sd7UIn8/s640/blogger-image-249045258.jpg)
```
Comment (ryansobol) [inline greydays/readme.md]: Prefix this line with `![]` to have GitHub display the image correctly. See `ryansobol/README.md`

Comment (greydays) [inline greydays/readme.md]: Ok. thanks!

**Comment by greydays** [comment]
updated readme.md

**Comment by ryansobol** [comment]
:+1:

**Pull request merged**